src/components/pages/Registration/RegisterFormComponents/StudentForm.js

```diff
@@ -41,6 +43,18 @@ export default function StudentForm({ studentArr, setStudentArr, idx }) {
     selectedAvatar === src ? 'selected-avatar' : '';
   // if the selected avatar matches the src of the clicked avatar, the class name will be set to selected avatar
 
+  const fileSelectedHandler = event => {
+    setSelectedFile(event.target.files[0]);
+  };
+
+  useEffect(() => {
+    const fd = new FormData();
```

Comment:
If possible, consider renaming this variable so its self-explanatory and future devs can easily tell what it stands for. Maybe, formData instead of fd?